### PR TITLE
README changes

### PR DESCRIPTION
**Pull request by geoffliu**
@adambard policy changes we discussed over email.

@vendethiel @Oire The README now explicitly mentions the two of you for French and Russian translations, is that OK? Also, any other natural language I missed? Oire, Ukrainian?

### CODE REVIEW

File: README.markdown [hunk 1]
```diff
@@ -1,32 +1,55 @@
...
+### Translations
+
+We encourage you to @-mention specific collaborators for translation-related
+pull requests of the following languages:
+
+* @geoffliu: Chinese (simplified and traditional), Italian
+* @oire: Russian
+* @vendethiel: French
+
+For all other languages, we leave the pull request open for a few weeks for the
```
Comment (vendethiel) [inline README.markdown]: "excluding english" maybe?

**Comment by vendethiel** [comment]
Maybe this should be `CONTRIBUTING.md`? So that it shows up in PR/issues.
The only other language I could take a look like would be spanish, but I don't feel I could quite review it :-).

File: README.markdown [hunk 2]
```diff
@@ -1,32 +1,55 @@
...
+section or doing a rewrite, add your name to the top of the page under the
+contributors section. We do not have strict policies on what counts as
+"significant," but please understand that cleanup patches or minor bug fixes do
+not count.
+
+### Small Fixes
+
+When opening issues and pull requests that are (or would be) only a few lines
+(usually < 5), we strongly encourage you to indicate as such in the title. We
+tend to prioritize small changes since we can get through more of them quickly.
```
Comment (adambard) [inline README.markdown]: I propose the `[Wee]` prefix for small changes.

**Comment by adambard** [comment]
Wow yeah CONTRIBUTING is totally a thing. How did we miss that? https://github.com/blog/1184-contributing-guidelines

**Comment by adambard** [comment]
I couldn't push to your repo, but I split that stuff if you want to merge that into this PR for further commentary: https://github.com/adambard/learnxinyminutes-docs/compare/new-contributing?expand=1

**Comment by adambard** [comment]
Upon further consideration, I think the contributing file ought to have the checklist of admonishments about `lang` and `filename` tags, tagging PRs, and whatnot, right at the top.

**Comment by geoffliu** [comment]
Oh huh yeah. I didn't know that's a thing. OK in that case I'll rework the wording, with the README encouraging people to contribute, and put the bullet points/checklists in the CONTRIBUTING.

**Comment by Oire** [comment]
@geoffliu, yes, add also Ukrainian for me, please :) Thanks!

**Comment by onebree** [comment]
@adambard I worked on the contributing file in #1753, but received no feedback.

**Comment by vendethiel** [comment]
Whoops, that's bad. I'm sorry that happened.

**Comment by onebree** [comment]
@vendethiel was that comment to me? I will update my PR today, so it can properly me reviewed and merged.

**Comment by vendethiel** [comment]
It was. I think your PR will be merged by @adambard, as he runs the project

**Comment by onebree** [comment]
Okay, thank you.

**Comment by geoffliu** [comment]
I... screwed up this PR.

**Comment by onebree** [comment]
@geoffliu If you would like, I could add the changes from your branch (before the mess-up today) into my PR #1753

**Comment by geoffliu** [comment]
Yes please, thanks!

Il giorno lun 15 feb 2016 alle ore 17:50 Hunter Stevens <
notifications@github.com> ha scritto:

**Comment by onebree** [comment]
@geoffliu Is this the only commit that you wanted to merge? Because everything else looks like merge commit overflow or something. (This way I can cherry pick the correct commit.)

https://github.com/geoffliu/learnxinyminutes-docs/commit/841c953c50df4d470e49fdf6630e175d9c48d9e8

**Comment by geoffliu** [comment]
Yup looks right.

Il giorno lun 15 feb 2016 alle ore 17:59 Hunter Stevens <
notifications@github.com> ha scritto:

**Comment by onebree** [comment]
@geoffliu my PR was accepted before I could put your commit in. I suggest making a new PR, off of master, with the following changes:

geoffliu@841c953

You then should not need huge merges like you had with this one.

**Comment by adambard** [comment]
The one time I try to be punctual!

**Pull request closed without merging**